[0m22:50:07.057402 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f96906a2c60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f96909796a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f969086a720>]}


============================== 22:50:07.060475 | 69a33839-94ad-45a8-a68d-11107a932d55 ==============================
[0m22:50:07.060475 [info ] [MainThread]: Running with dbt=1.7.7
[0m22:50:07.062047 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:50:27.746695 [debug] [MainThread]: Starter project path: /usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/include/starter_project
[0m22:50:27.933531 [info ] [MainThread]: 
Your new dbt project "ad_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m22:50:27.934978 [info ] [MainThread]: Setting up your profile.
[0m22:50:50.724284 [error] [MainThread]: Encountered an error:

[0m22:50:50.728522 [error] [MainThread]: Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/cli/requires.py", line 91, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/cli/requires.py", line 76, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/cli/main.py", line 515, in init
    results = task.run()
              ^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/task/init.py", line 344, in run
    self.setup_profile(profile_name)
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/task/init.py", line 264, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/task/init.py", line 181, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/task/init.py", line 165, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/task/init.py", line 118, in generate_target_from_input
    target = self.generate_target_from_input(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/dbt/task/init.py", line 131, in generate_target_from_input
    target[key] = click.prompt(
                  ^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/exit/lib/python3.12/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m22:50:50.730309 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 43.67989, "process_user_time": 1.619946, "process_kernel_time": 0.168557, "process_mem_max_rss": "122008", "process_in_blocks": "240", "command_success": false, "process_out_blocks": "0"}
[0m22:50:50.731596 [debug] [MainThread]: Command `dbt init` failed at 22:50:50.731445 after 43.68 seconds
[0m22:50:50.732677 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f96909796a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f968ebb0110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f968ee497f0>]}
[0m22:50:50.733794 [debug] [MainThread]: Flushing usage events
